web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -402,12 +402,11 @@ function joinup_core_post_update_update_update_eira_terms() {
   $filename = DRUPAL_ROOT . '/../resources/fixtures/EIRA_SKOS.rdf';
   $graph->parseFile($filename);
 
-  // Copied over from \Drupal\rdf_entity\Entity\RdfEntitySparqlStorage::insert.
-  $graph_uri = SparqlArg::uri($graph_uri);
-  $query = "INSERT DATA INTO $graph_uri {\n";
-  $query .= $graph->serialise('ntriples') . "\n";
-  $query .= '}';
-  $connection->update($query);
+  $sparql_connection = Database::getConnection('default', 'sparql_default');
```

Comment:
@sandervd @claudiu-cristea, given that the GraphStore is a class of an external library, do you think that it might be a good idea to have sparql_storage have a static method that returns such a graphstore as below? 
Because initially it might seem good though it might give the idea that we are encouraging users to use custom queries instead of the API.